Formula/g/gopls.rb

```diff
@@ -24,7 +25,7 @@ class Gopls < Formula
 
   def install
     cd "gopls" do
-      system "go", "build", *std_go_args(ldflags: "-s -w")
+      system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=v#{version}")
```

Comment:
Prefixing with `v` mirrors what you get if you do `go install golang.org/x/tools/gopls@latest` (which is the documented installation procedure).